data/base-hunting.yaml

```diff
@@ -966,6 +966,13 @@ hunting_zones:
   - 3073
   - 3072
   - 3070
+  #https://elanthipedia.play.net/Giant_thicket_viper
```

Comment:
💡 It would be nice to have an approximate range of ranks in here too (e.g. line 976)